pandas/tests/indexes/multi/test_contains.py

```diff
@@ -58,6 +58,25 @@ def test_isin():
     assert result.dtype == np.bool_
 
 
+@pytest.mark.parametrize(['values', 'badix'],
+                         [([('foo',), ('bar', 3), ('quux',)], 0),
+                          ([('foo', 2), ('bar', 3, 2), ('quux', 4)], 1)])
+def test_isin_values_raises(values, badix):
+    print(values)
```

Comment:
remove the print statements

Reply:
whoops, left those in from when I was figuring out the parameter syntax